### ISSUE TITLE
Interactive tool windows lack BitmapImageMoniker

### ISSUE THREAD

**Issue by amcasey** [dotnet/roslyn#6218]
Need to set ```ToolWindowPane.BitmapImageMoniker``` in ```OnToolWindowCreated```.

**Comment by dpoeschl** [comment]
Is this the thing causing the resized window to not have an icon?

**Comment by amcasey** [comment]
Yes.  It looks straightforward except for some plumbing required to share between C#, VB, and Python.

**Comment by amcasey** [comment]
We could expose it through the evaluator, but that seems like a hack.  We'll do it properly in 1.2 (at which point the monikers will also become available).

**Comment by CyrusNajmabadi** [comment]
Closing out due to lack of movement.
